src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -41,230 +41,207 @@ const defaultProps = {
-    }
-
-    render() {
-        const policy = this.props.policy;
-        const policyName = lodashGet(this.props.policy, 'name', '');
-        const hasMembersError = PolicyUtils.hasPolicyMemberError(this.props.policyMemberList);
-        const hasGeneralSettingsError = !_.isEmpty(lodashGet(this.props.policy, 'errorFields.generalSettings', {}))
-            || !_.isEmpty(lodashGet(this.props.policy, 'errorFields.avatar', {}));
-        const hasCustomUnitsError = PolicyUtils.hasCustomUnitsError(this.props.policy);
-        const menuItems = [
-            {
-                translationKey: 'workspace.common.settings',
-                icon: Expensicons.Gear,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceSettingsRoute(policy.id)),
-                brickRoadIndicator: hasGeneralSettingsError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : '',
-            },
-            {
-                translationKey: 'workspace.common.card',
-                icon: Expensicons.ExpensifyCard,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceCardRoute(policy.id)),
-            },
-            {
-                translationKey: 'workspace.common.reimburse',
-                icon: Expensicons.Receipt,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceReimburseRoute(policy.id)),
-                error: hasCustomUnitsError,
-            },
-            {
-                translationKey: 'workspace.common.bills',
-                icon: Expensicons.Bill,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceBillsRoute(policy.id)),
-            },
-            {
-                translationKey: 'workspace.common.invoices',
-                icon: Expensicons.Invoice,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceInvoicesRoute(policy.id)),
-            },
-            {
-                translationKey: 'workspace.common.travel',
-                icon: Expensicons.Luggage,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceTravelRoute(policy.id)),
-            },
-            {
-                translationKey: 'workspace.common.members',
-                icon: Expensicons.Users,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceMembersRoute(policy.id)),
-                brickRoadIndicator: hasMembersError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : '',
-            },
-            {
-                translationKey: 'workspace.common.bankAccount',
-                icon: Expensicons.Bank,
-                action: () => ReimbursementAccount.navigateToBankAccountRoute(policy.id),
-            },
-        ];
-
-        return (
-            <ScreenWrapper includeSafeAreaPaddingBottom={false}>
-                {({safeAreaPaddingBottomStyle}) => (
-                    <FullPageNotFoundView
-                        shouldShow={_.isEmpty(this.props.policy)}
+    }, [props.reports, props.policy]);
+
+    const policy = props.policy;
```

Comment:
Since you're creating a local variable for `policy`, let's move it up and use it on line 72 where we're currently using `props.policy`?